crates/builder/src/generate/generator.rs

```diff
@@ -51,14 +45,14 @@ impl Generator {
         Ok(())
     }
 
-    pub fn text(&self, module: &str, assets: &[Asset]) -> String {
+    pub fn text(&self, assets: &[Asset]) -> String {
         let constants = self.constants(assets);
         let matching = self.match_list(assets);
         format!(
             r#"
 /// This is a generated file. Do not edit. It is updated depending on the build profile used (i.e. dev, release).
-/// Instead it should be included with an include! macro: `include!("../gen/{module}.rs");`
-pub mod {module} {{
+/// Instead it should be included with an include! macro: `include!("../gen/{MODULE_FILE}.rs");`
```

Comment:
MODULE_FILE and MODULE cannot be hard-coded because they should be named according to the assembly name. If there is no assembly, then it could fall back to "assets"

Reply:
but we removed the assembly name, so it will always be "assets"

Reply:
Hm. There needs to be a way to create more than one localisation (esp. generate different variants) Well, it can be solved by making the generated file path configurable with something like:

```toml
[[package.metadata.prebuild.localized]]
profile = ["release"]
path = "img/apple_store"
file-extension = "svg"
out = { folder = "static/badge", checksum = true, brotli = true, gzip = true }
# where to write the code
generate-code = "gen/mobile.rs"
```